src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -187,6 +191,34 @@ const ResearchProfile = () => {
           </Box>
         )}
 
+        {(person.contactDetails?.email || person.contactDetails?.telephone || person.contactDetails?.webPage) && (
+          <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.25rem', mt: '0.75rem' }}>
+            <Typography fontWeight="bold">{t('my_page.my_profile.contact_information')}</Typography>
+            {person.contactDetails?.email && (
+              <Box sx={{ display: 'flex', gap: '0.5rem' }}>
+                <MailOutlineIcon />
+                <Link href={`mailto:${person.contactDetails?.email}`}>
```

Comment:
sjekker at denne ikke er tom på linje 197
```suggestion
                <Link href={`mailto:${person.contactDetails.email}`}>
```